tutorials/editor/command_line_tutorial.rst

```diff
@@ -412,6 +415,13 @@ preset name and output path, with the file extension, instead of
 ``--export-release`` or ``--export-debug``. The output path extension determines
 the package's format, either PCK or ZIP.
 
+To export a PCK file for something like a patch, mod or DLC, use the
```

Comment:
```suggestion
To export a PCK file for something like a patch, mod, or DLC, use the
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).